core-it-suite/src/test/resources-filtered/bootstrap.txt

```diff
@@ -79,13 +79,15 @@ org.apache.maven.plugins:maven-dependency-plugin:3.3.0
 org.apache.maven.plugins:maven-deploy-plugin:${stubPluginVersion}
 org.apache.maven.plugins:maven-deploy-plugin:2.7
 org.apache.maven.plugins:maven-deploy-plugin:3.0.0
+org.apache.maven.plugins:maven-deploy-plugin:3.0.0-M2
```

Comment:
these will not be needed once https://github.com/apache/maven-integration-testing/pull/230 merged

Reply:
rebased